packages/manager-components/src/components/templates/delete-modal/delete-modal.component.tsx

```diff
@@ -19,10 +20,10 @@ import {
   ODS_TEXT_LEVEL,
   ODS_TEXT_SIZE,
 } from '@ovhcloud/ods-components';
-import { ApiError } from '@ovh-ux/manager-core-api';
-import { handleClick } from '@/utils/ods-utils';
-import { FormField } from '../FormField';
-import { terminateValue } from './DeleteModal.constants';
+import { handleClick } from '../../../utils/click-utils';
+import './translations/translations';
+
+const terminateValue = 'TERMINATE';
```

Comment:
Maybe put this in the props to change it, if needed to change it from the component

Reply:
Thanks done